### PR TITLE
chore: change deploy to new internal deploy functionality

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Change deploy to new internal deploy functionality

### CODE REVIEW

File: .circleci/config/jobs/deploy_to_dev.yml [hunk 1]
```diff
@@ -14,10 +14,28 @@ steps:
...
-            -X POST -d "{\"docker_sha\":\"${LATEST_TAG}\", \
-                          \"commit_hash\":\"${CIRCLE_SHA1}\"}" \
-            https://my.dev.snyk.io/${DEV_DEPLOY_TOKEN}
+        git clone https://$GH_TOKEN@github.com/snyk/$KUBERNETES_MONITOR_DEPLOYER_REPO.git
+        mkdir -p $KUBERNETES_MONITOR_DEPLOYER_REPO/helm/values/
+        cat >$KUBERNETES_MONITOR_DEPLOYER_REPO/helm/values/pre-prod-mt.yaml <<EOF
+        clusterName: "Development cluster"
+        integrationApi: https://kubernetes-upstream.dev.snyk.io
+        log_level: "DEBUG"
+        policyOrgs: {$POLICY_ORG_PRE_PROD}
```
Comment (ivanstanev) [inline .circleci/config/jobs/deploy_to_dev.yml:23]: Nit: `${`

Comment (ivanstanev) [reply .circleci/config/jobs/deploy_to_dev.yml:23]: Actually wait, this is Helm array syntax?

Comment (dkontorovskyy) [reply .circleci/config/jobs/deploy_to_dev.yml:23]: It's an object that looks like `{uuid}`, so I believe it's correct here

**Review by ivanstanev — APPROVED** [review]

File: .circleci/config/jobs/deploy_to_dev.yml [hunk 2]
```diff
@@ -14,10 +14,31 @@ steps:
         LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}` &&
         LATEST_TAG=${LATEST_TAG_WITH_V:1}-approved &&
         ./scripts/slack/notify_deploy.py $LATEST_TAG dev &&
-        curl -s --fail --show-error -i -H "Accept: application/json" -H "Content-Type: application/json" \
-            -X POST -d "{\"docker_sha\":\"${LATEST_TAG}\", \
-                          \"commit_hash\":\"${CIRCLE_SHA1}\"}" \
-            https://my.dev.snyk.io/${DEV_DEPLOY_TOKEN}
+        git clone https://$GH_TOKEN@github.com/snyk/$KUBERNETES_MONITOR_DEPLOYER_REPO.git
```
Comment (dkontorovskyy) [inline .circleci/config/jobs/deploy_to_dev.yml:17]: 🛑 Remove the content of `helm/` folder in this repo before merging 🛑

**Review by kat1906 — APPROVED** [review]

**Pull request merged**